.github/workflows/continuous-integration-lint-php.yml

```diff
@@ -34,9 +34,6 @@ jobs:
             ${{ runner.os }}-composer-
             ${{ runner.os }}-
 
-      - name: Validate composer.json
```

Comment:
Is validation failing with the more complex setup now? Maybe worth fixing?

Reply:
It's a super minor thing.

[`mcaskill/composer-exclude-files`](https://packagist.org/packages/mcaskill/composer-exclude-files) requires setting `auttoload.exclude-from-files` but Composer doesn't know that option and complains about it. I didn't see a way to circumvent that error in Composer's validate command as the list of allowed options is hardcoded.

Reply:
Ah, that sounds like a bug in the plugin. Composer plugins should add their configuration options to the `extra` key.

Reply:
Created upstream issue: https://github.com/mcaskill/composer-plugin-exclude-files/issues/8

Reply:
Awesome, thanks. Just subscribed 👍